.github/workflows/no-prs-to-translation.yml

```
@@ -6,6 +6,8 @@ on:
     paths:
       - 'curriculum/challenges/**'
       - '!curriculum/challenges/english/**'
+      - 'client/i18n/locales/**'
```

```suggestion
      - 'client/i18n/locales/**/intro.json'
      - 'client/i18n/locales/**/translations.json'
```
Those are the only two that are handled by Crowdin, right?

---

Oops.  Yes, I meant to do that but for some reason just mimicked the previous two lines.